lib/candy_check/play_store/acknowledger.rb

```diff
@@ -0,0 +1,19 @@
+module CandyCheck
+  module PlayStore
+    class Acknowledger
+      def initialize(authorization:)
+        @authorization = authorization
+      end
+
+      def acknowledge_product_purchase(package_name:, product_id:, token:)
+        acknowledger = CandyCheck::PlayStore::ProductAcknowledgements::Acknowledgement.new(
+          package_name: package_name,
+          product_id: product_id,
+          token: token,
+          authorization: @authorization,
+        )
+        acknowledger.call!
```

Comment:
Like I say [here](https://github.com/Badiapp/candy_check/pull/1/files#r444000801), it is weird that the `Acknowledger` does not have the logic to acknowledge things itself.

This class does nothing basically 🤔

Reply:
I agree that is a wrapper of `acknowledgement` as [Verifier](https://github.com/Badiapp/candy_check/blob/feature/add-server-ack-for-consumable/lib/candy_check/play_store/verifier.rb#L5) of [ProductVerification](https://github.com/Badiapp/candy_check/blob/feature/add-server-ack-for-consumable/lib/candy_check/play_store/product_purchases/product_verification.rb#L6). I think the idea why is done this way is to have a simpler API.

Reply:
Just saw that this "dumb" class is how the gem operates with the ["verifier"](https://github.com/Badiapp/candy_check/blob/master/lib/candy_check/play_store/verifier.rb)

Reply:
Go ahead!